<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1488</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QCustomPlot" name="xSatellite" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="ySatellite" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="VxSatellite" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="VySatellite" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>480</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1488</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
